tutorials/tcpip/pcap-driver/main.c

```diff
@@ -252,6 +281,10 @@ int main(int argc, char *argv[]) {
          &mif.mac[2], &mif.mac[3], &mif.mac[4], &mif.mac[5]);
   mg_tcpip_init(&mgr, &mif);
 
+  // Call order is important: call after mg_tcpip_init()
```

Comment:
Let's wrap these two lines in a conditional, like `#if defined(DISABLE_DHCP)`
Also, the if_ev_handler() function.